packages/context-entries/lib/Keys.ts

```diff
@@ -39,6 +39,10 @@ export const KeysHttp = {
    * Fetch function implementation.
    */
   fetch: new ActionContextKey<typeof fetch>('@comunica/bus-http:fetch'),
+  /**
+   * HTTP request timeout in milliseconds.
+   */
+  httpTimeout: new ActionContextKey<number>('@comunica/bus-http:http-timeout'),
```

Comment:
We'll also want to expose this via the CLI.
This can be done here:
- https://github.com/comunica/comunica/blob/master/packages/actor-init-query/lib/ActorInitQueryBase.ts#L101-L118
- https://github.com/comunica/comunica/blob/master/packages/actor-init-query/lib/cli/CliArgsHandlerBase.ts#L78

Reply:
Sure!